pyproject.toml

```diff
@@ -167,14 +157,6 @@ ignore = [
 
     # `subprocess` call: check for execution of untrusted input
     "S603",
-    # Prefer `next(iter(info_dicts))` over single element slice
-    "RUF015",
```

Comment:
I agree that `next(generator expression)` is better than `[list comprehension][0]`, and that does seem to be where most of the changes are. I'm less convinced about `list(x)[0]` (although that ought to be `tuple(x)[0]` in my opinion).

Are switching this because we have decided that we like this style after all, or just because we want to follow the ruff defaults more closely?

Reply:
I think the ruff rule makes sense here, although my original idea was only to remove as many ignored rules as possible to stay consistent with the community. I will keep the next generator then.